_posts/2021-05-20-CRA-tutorial.md

```diff
@@ -1,33 +1,28 @@
 ---
-title: "Installing CloudForest, and utilizing the Cipres Rest API to infer trees"
+title: "Utilizing the Cipres Rest API to infer trees"
 date: 20-May-2021
 categories:
-  - Getting Started
+  - Analyzing Data
 tags:
   - Jekyll
   - update
 youtubeId: iKRTPDBHynw
 ---
 {% include youtubePlayer.html id=page.youtubeId %}
 
-##### 1. Install and run Docker
-Installation directions vary by operating system. Detailed instructions can be found [here](https://docs.docker.com/get-docker/). Once downloaded and run, Docker runs in the background. 
-  
-***NOTE:*** *If using Windows, it is recommended to download and install a ***WSL 2*** linux instance on your computer for easier integration with Docker upon install. More details on installing ***WSL 2*** can be found [here](https://www.omgubuntu.co.uk/how-to-install-wsl2-on-windows-10).*  
-  
-##### 2. Create a Cipres Rest API account
+##### 1. Create a Cipres Rest API account
 Registration can be completed [here](https://www.phylo.org/restusers/register.action). After your account is created, no further account configuration is necessary. *The credentials you create in this step will be used by CloudForest for remote job submission.*
 
 ***NOTE:*** *Steps 3-4 are necessary in the Beta release of CloudForest, but will not be present in the final release. Instead, CloudForest will be packaged for easy installation and running*
 
-##### 3. Clone and Navigate to the CloudForestDocker repository
+##### 2. Clone and Navigate to the CloudForestDocker repository
 Once Docker is installed and is running in the background, navigate to your preferred directory from a Unix command line and run the following commands:  
 ```
 $ git clone -b beta_testing https://github.com/TreeScaper/CloudforestDocker.git
 $ cd CloudforestDocker  
 ```
 
-##### 4. Run CloudForest
+##### 3. Run CloudForest
 From within the CloudForestDocker directory, run the following command:  
 ```

```

Comment:
If we're creating a separate install tutorial, these lines should probably be taken out right?